backend/app/core/validators/gender_assumption_bias.py

```
@@ -0,0 +1,78 @@
+from app.core.config import Settings
+from app.core.enum import BiasCategories
+
+@register_validator(name="gender-assumption-bias", data_type="string")
+class GenderAssumptionBias(Validator):
+    """
+    Validate text for the presence of gender assumption in LLM generated outputs.
+    """
+
+    def __init__(
+        self, 
+        categories: Optional[List[BiasCategories]] = [BiasCategories.All],
```

Ruff (0.14.8)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)